Doc/tutorial/inputoutput.rst

```diff
@@ -322,6 +322,10 @@ equivalent :keyword:`try`\ -\ :keyword:`finally` blocks::
 
     >>> with open('workfile') as f:
     ...     read_data = f.read()
+
+After the :keyword:`with` statement, we can see that the file has been
+automatically closed::
+
     >>> f.closed
```

Comment:
This might be clearer as just an inline comment in order to keep the example all together in one block.  So, before the f.closed(), how about:
```
>>> # We can check that the file has been automatically closed.
```